Samples~/AvatarCreatorSamples/AvatarCreatorWizard/Scripts/UI/ReloadPreviouslyEditedAvatar.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using ReadyPlayerMe.AvatarCreator;
+using ReadyPlayerMe.Core;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace ReadyPlayerMe.Samples.AvatarCreatorWizard
+{
+    public class ReloadPreviouslyEditedAvatar : MonoBehaviour
+    {
+        private const string TAG = nameof(ReloadPreviouslyEditedAvatar);
+
+        [SerializeField] private Button continueEditingAvatar;
+        [SerializeField] private Button cancelEditingAvatar;
+
+        private string avatarId;
+        public UnityEvent<string> onSaveAvatar;
```

Comment:
Rider warns about using capital letters for event names however I couldn't have it auto-enforce this convention. But it should be a capital first letter for public Unity events or actions or properties to keep it consistent across our SDK. 
```suggestion
        public UnityEvent<string> OnSaveAvatar;
```
Also I think this event should be renamed to something more like OnContinueEditingAvatar or something